CMakeLists.txt

```diff
@@ -68,6 +68,11 @@ else ()
 endif ()
 option(GO_SEPARATE_LINUX_PACKAGES "Generate separate linux packages for resources and demo" OFF)
 
+# Add ASIO variable
+if(RTAUDIO_USE_ASIO)
```

Comment:
Could you move this below in the file, before ``if(RTAUDIO_USE_ASIO AND NOT ASIO_SDK_DIR)``? It'd better to have  all asio-related things at one place.

Reply:
Ok, I looked quickly but didn't find them.

Reply:
And it still doesn't work on linux.

Try to use ``if(WIN32 AND RTAUDIO_USE_ASIO)``